README.md

```diff
@@ -6,13 +6,13 @@ The Angular frontend and associated Scala API for Workflow.
 In order to run, workflow-frontend needs to talk to the CODE workflow datastore. It does this via an SSH tunnel to a 
 workflow-frontend CODE instance.
 
-- Make sure that you are running the right version of nodejs, should be v0.10.45. 
-- Run the install script `./setup.sh` 
+- Make sure that you are running the right version of nodejs, (tested and working with v6.1.0). 
+- Run the install script `./scripts/setup.sh`
```

Comment:
I've tend to agree but as this is used on teamcity too, it'd break on there. Maybe an improvement in the future would be to push all this to teamcity build steps and use this file solely for local dev.